lib/tls.js

```diff
@@ -15,9 +15,9 @@ exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
 exports.DEFAULT_CIPHERS =
     // TLS 1.2
-    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' +
+    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:' +
     // TLS 1.0
-    'RC4:HIGH:!MD5:!aNULL';
+    ':HIGH:!RC4:!MD5:!aNULL';
```

Comment:
@bnoordhuis it doesn't happen unless we bump minor OpenSSL version. I'm -1 for query.